projects/plugins/jetpack/_inc/client/sharing/publicize.jsx

```diff
@@ -179,6 +179,9 @@ export const Publicize = withModuleSettingsFormHelpers(
 					) }
 
 					{ isActive && configCard() }
+					{ window.Initial_State.socialInitialState.useAdminUiV1 ? (
```

Comment:
Should we add a helper in this change so we can just get the value from the publicize config hook?

Exaple: fbhepr%2Skers%2Swrgcnpx%2Scebwrpgf%2Swf%2Qcnpxntrf%2Schoyvpvmr%2Qpbzcbaragf%2Sfep%2Subbxf%2Shfr%2Qchoyvpvmr%2Qpbasvt%2Svaqrk.wf%3Se%3Q5r00q081%2395-og

Reply:
This is a class component, we can't use that hook here.

Reply:
That's annoying! Should it be part of the social store? It's hopefully not going to be longlived, so perhaps accessing it directly on the window object is fine.